L127-cpp-spiffe-root-provider.md

```diff
@@ -50,4 +50,7 @@ FileWatcherCertificateProvider(const std::string& private_key_path,
 ```
 
 ### Other Providers
-This proposal _only_ aims to support file-based SPIFFE Bundle Maps via the file watcher providers. The `StaticDataCertificateProvider` structure is left as future work. This will involve broadening the API surface to expose a type for the SPIFFE bundle map.
\ No newline at end of file
+This proposal _only_ aims to support file-based SPIFFE Bundle Maps via the file watcher providers. The `StaticDataCertificateProvider` structure is left as future work. This will involve broadening the API surface to expose a type for the SPIFFE bundle map.
+
+## Implementation
+PR will be linked when created.
```

Comment:
I think you can list all of the PRs you've been working on that are directly related to this effort:
- https://github.com/grpc/grpc/pull/39148
- https://github.com/grpc/grpc/pull/39445
- https://github.com/grpc/grpc/pull/39708

You can add to the list as you finish the remaining work.